vars/jenkinsApi.groovy

```diff
@@ -0,0 +1,21 @@
+def getSteps() {
+  def url = "${env.BUILD_URL}api/json?tree=actions[nodes[iconColor,running,displayName,id,parents]]"
```

Comment:
Creating URLs with proper tools looks like this:

```groovy
def builder = new URIBuilder(env.BUILD_URL)
def url = builder
    .setPathSegments(builder.getPathSegments() + ['api', 'json'])
    .setParameter('tree', 'actions[nodes[iconColor,running,displayName,id,parents]]')
    .toString()
```

and actually produces incorrectly-formatted URLs, e.g.
`https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/34782//api/json?tree=actions%5Bnodes%5BiconColor%2Crunning%2CdisplayName%2Cid%2Cparents%5D%5D`

(notice the double `//`)